enterprise/runtime/neole/src/main/java/org/neo4j/internal/store/prototype/neole/PropertyCursor.java

```diff
@@ -83,6 +84,12 @@
      *                                                                                 0 means 64, other values "normal"
      *            [xxxx,xxxx] [xxxx,xxxx] [xxxx,    ] [    ,    ] payload(+ maybe in next block) (0xFFFF_FF00_0000_0000)
      *                                                            bits are densely packed, bytes torn across blocks
+     * POINT:     [    ,    ] [    ,    ] [    ,    ] [    ,    ] [xxxx,type][K][K][K] geometry subtype
```

Comment:
please move this format description to a place where that format is defined as well and reference this class or vica versa in javadoc

Reply:
We are actually about to nuke NeoLE from the neo4j code base, so this is definitely not the future-proof place to document format changes. Maybe somewhere around the `PropertyRecordFormat`?

Reply:
We should move all the format descriptions here to a better place. We only added the point format here because it was the only place where the formats are currently described in a reasonable way. Perhaps that move is done as part of the 'nuke neole' work?

Reply:
Moved to PropertyStore